### PR TITLE
Secure URLs (1 of 3)

### PR DESCRIPTION
**Pull request by vszakats**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/jhiccup.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Jhiccup < Formula
   desc "Measure pauses and stalls of an app's Java runtime platform"
-  homepage "http://www.jhiccup.com"
-  url "http://www.azulsystems.com/sites/default/files/images/jHiccup.1.3.7.zip"
+  homepage "http://www.jhiccup.com/"
+  url "https://www.azulsystems.com/sites/default/files/images/jHiccup.1.3.7.zip"
```
Comment (ilovezfs) [inline Formula/jhiccup.rb]: This is 404

Comment (vszakats) [inline Formula/jhiccup.rb]: Strange, works fine here:
```
$ curl -LI https://www.azulsystems.com/sites/default/files/images/jHiccup.1.3.7.zip
HTTP/1.1 200 OK
Date: Thu, 13 Jul 2017 16:06:33 GMT
Server: Apache/2.4.25 (Unix) OpenSSL/1.0.1e-fips mod_bwlimited/1.4
Last-Modified: Fri, 03 Jan 2014 18:56:08 GMT
ETag: "2e80f12-2c9a1-4ef15767b8200"
Accept-Ranges: bytes
Content-Length: 182689
Cache-Control: max-age=1209600
Expires: Thu, 27 Jul 2017 16:06:33 GMT
Content-Type: application/zip
```

The plain HTTP doesn't:
```
$ curl -LI http://www.azulsystems.com/sites/default/files/images/jHiccup.1.3.7.zip
HTTP/1.1 302 Found
Date: Thu, 13 Jul 2017 16:07:29 GMT
Server: Apache/2.4.25 (Unix) OpenSSL/1.0.1e-fips mod_bwlimited/1.4
Location: https://www.azul.com/sites/default/files/images/jHiccup.1.3.7.zip
Content-Type: text/html; charset=iso-8859-1

HTTP/1.1 404 Not Found
Date: Thu, 13 Jul 2017 16:07:31 GMT
Server: Apache/2.4.25 (Unix) OpenSSL/1.0.1e-fips mod_bwlimited/1.4
X-Powered-By: PHP/5.3.29
P3P: CP="NOI"
Pragma: no-cache
Expires: Wed, 11 Jan 1984 05:00:00 GMT
Cache-Control: no-cache, must-revalidate, max-age=0
Link: <https://www.azul.com/wp-json/>; rel="https://api.w.org/"
Set-Cookie: PHPSESSID=c968fe0ce1414d868c39404355bfaf7c; path=/
Content-Type: text/html; charset=UTF-8
```

Comment (ilovezfs) [inline Formula/jhiccup.rb]: looks like there is a new version, so this one probably belongs in a separate PR

Comment (ilovezfs) [inline Formula/jhiccup.rb]: Actual page says "404 - Page Not found"

Comment (ilovezfs) [inline Formula/jhiccup.rb]: New version is here: https://www.azul.com/jhiccup/

Comment (vszakats) [inline Formula/jhiccup.rb]: Let me remove this one from here.

**Comment by ilovezfs** [comment]
All were identified by brew audit except the one that is no longer available, and the one in the ruby comment, of course.

**Comment by ilovezfs** [comment]
:lock:

**Comment by vszakats** [comment]
Thank you @ilovezfs!
